docs/user_guide/Initial_Server_Configuration/Prevent_Lockout_on_Puppetserver.inc

```diff
@@ -61,109 +58,33 @@ This example creates a ``local_system_access`` :term:`Puppet module` in the
-
-      umask 022
-
-#. Create a ``local_system_access`` puppet module directory and change to the directory.
-
-   .. code-block:: sh
-
-      cd /etc/puppetlabs/code/environments/production/modules
-      mkdir -p local_system_access/manifests
-      cd local_system_access
-
-#. Add the following to a new ``manifests/local_user.pp`` file to enable ``sudo su - root``
-   and allow ``ssh`` access for the user you created/selected:
-
-   .. code-block:: ruby
-
-      class local_system_access::local_user (
-        Boolean $pam = simplib::lookup('simp_options::pam', { 'default_value' => false }),
-      ) {
-
-        sudo::user_specification { 'default_userx':
-          user_list => ['userx'],
-          runas     => 'root',
-          # ONLY NEEDED IF YOUR USER DOES NOT USE A PASSWORD
-          passwd    => false,
-          cmnd      => ['/bin/su root', '/bin/su - root']
-        }
-
-        if $pam {
-          include 'pam'
-
-          pam::access::rule { 'allow_userx':
-            users   => ['userx'],
-            origins => ['ALL'],
-            comment => 'Local user for lockout prevention'
-          }
-        }
-      }
-
-#. Add the following to a new ``metadata.json`` file to enable proper
-   recognition of your module by the puppet server:
-
-   .. code-block:: yaml
-
-      {
-        "name": "local_system_access",
-        "version": "0.0.1",
-        "author": "Your name or group here",
-        "summary": "Configures Local User for sudo access",
-        "license": "Apache-2.0",
-        "source": "Your gitlab url or local",
-        "dependencies": [
-          {
-            "name": "simp/pam"
-          },
-          {
-            "name": "simp/simplib"
-          },
-          {
-            "name": "simp/sudo"
-          }
-        ]
-      }
-
-   (See the `Puppet metadata documentation`_ for more information on metadata.json files.)
-
-#. Make sure the permissions are correct on the module:
-
-   .. code-block:: bash
-
-      chown -R root:puppet $PWD
-      chmod -R g+rX $PWD
-
-#. Add the module to the SIMP server's host YAML file class list:
-
-   .. code-block:: bash
-
-      cd /etc/puppetlabs/code/environments/production/data/hosts
-
-   Add ``local_system_access::local_user`` to the ``simp::classes:`` array in ``<SIMP server FQDN>.yaml``
-
-   .. code-block:: yaml
-
-      simp::classes:
-        - local_system_access::local_user
-        # Do NOT remove other items in this array
-        # Make sure your whitespace lines up (spaces, not tabs)
-
-#. Add the ``local_system_access`` module to the ``Puppetfile`` in the ``production`` environment:
-
-   Edit ``/etc/puppetlabs/code/environments/production/Puppetfile``, and add the
-   following line under the section that says *"Add your own Puppet modules here"*
-
-   .. code-block:: ruby
-
-      mod 'local_system_access', :local => true
-
+#. Run ``cd /etc/puppetlabs/code/environments/production/data``
+
+#. Add the following to ``default.yaml``
```

Comment:
Shouldn't this be put in the puppet server yaml?  I thought this was only problem on the puppet server.

Reply:
Other than that I copied and pasted the content and added the user and it worked fine.

Reply:
> Shouldn't this be put in the puppet server yaml? I thought this was only problem on the puppet server.

Unless they've been added to `pam::access` somewhere else, the lockout would affect local user access on agents, too.